content/_help/specific-agencies/trusted-traveler-programs._es.md

```diff
@@ -34,10 +34,16 @@ La cuenta de Login.gov es únicamente para iniciar sesión de forma segura. Su c
 Login.gov solo puede responder las preguntas sobre el proceso de inicio de sesión y la creación de una cuenta de Login.gov.
 
 Contacte directamente con los [programas para viajeros de confianza](https://help.cbp.gov/s/questions?language=es) si tiene preguntas acerca de:
-* Estado o cambios de su solicitud
-* Cumplimiento de requisitos
-* Programación o cambio de citas
-* Otras consideraciones afines
+
+* Solicitudes
+  * [Estado de la solicitud](https://help.cbp.gov/s/all-ttp-articles?language=es)
```

Comment:
This goes to the list of FAQs rather than the specific article that's linked in English. It looks like there may not be a Spanish translation of that page, but I'd still expect it to link to the article, even if it shows in English.

```suggestion
  * [Estado de la solicitud](https://help.cbp.gov/s/article/Article-1886?language=es)
```